kura/distrib/pom.xml

```diff
@@ -33,40 +33,74 @@
         <module>docker-base</module>
         <module>docker-alpine</module>
         <module>docker-ubi8</module>
         -->
     </modules>
-    
+
     <properties>
         <tycho-version>4.0.11</tycho-version>
-        <jdeb-version>1.12</jdeb-version>
+        <jdeb-version>1.14</jdeb-version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <kura.basedir>${project.basedir}</kura.basedir>
         <kura.build.version>${maven.build.timestamp}</kura.build.version>
 
         <kura.skip.can>true</kura.skip.can>
 
-        <!-- used for RPM uninstallation -->
-        <kura.install.link>/opt/eclipse/kura</kura.install.link>
+        <kura.install.dir>/opt/eclipse</kura.install.dir>
+        <kura.symlink>kura</kura.symlink>
 
         <!-- properties used for repository upload on artifactory -->
         <kura.repo.distribution>kura-6</kura.repo.distribution>
         <kura.repo.module>base</kura.repo.module>
 
         <org.eclipse.equinox.launcher.version>1.6.900</org.eclipse.equinox.launcher.version>
-        <kura.version>${project.version}</kura.version>
+
+        <kura.version>KURA_${project.version}</kura.version>
+        <kura.marketplace.compatibility.version>${kura.version}</kura.marketplace.compatibility.version>
+        <kura.home>${kura.install.dir}/${kura.symlink}</kura.home>
+        <kura.framework.config>${kura.home}/framework</kura.framework.config>
+        <kura.user.config>${kura.home}/user</kura.user.config>
+        <kura.plugins>${kura.home}/plugins</kura.plugins>
+        <kura.packages>${kura.home}/packages</kura.packages>
+        <kura.data>${kura.home}/data</kura.data>
+        <kura.snapshots>${kura.home}/snapshots</kura.snapshots>
+        <kura.style.dir>${kura.home}/console.skin</kura.style.dir>
+
+        <kura.project>generic-${project.name}</kura.project>
+        <kura.have.net.admin>false</kura.have.net.admin>
+
+        <skip.resource.copy>true</skip.resource.copy>
+
+        <device.name>device_name</device.name>
+        <os.distribution>Linux</os.distribution>
+        <os.distribution.version>N/A</os.distribution.version>
+        <kura.primary.network.interface>kura.primary.network.interface=eth0</kura.primary.network.interface>
+        <kura.serialNumber.provider>cat /proc/cpuinfo | grep Serial | cut -d ' ' -f 2</kura.serialNumber.provider>
+        <kura.serialNumber></kura.serialNumber>
+        <deb.name>kura-core</deb.name>
+        <deb.dependencies>
+            setserial, zip, gzip, unzip, procps, usbutils, socat, gawk, sed, inetutils-telnet,
+            polkit | policykit-1, ssh | openssh, openssl, busybox, cron, logrotate, libpam-modules,
+            python3, openjdk-17-jre-headless | temurin-17-jdk, dos2unix, libtirpc3, chrony
+        </deb.dependencies>
+        <deb.recommendations></deb.recommendations>
+        <deb.provides></deb.provides>
+        <deb.architecture>unset</deb.architecture>
```

Comment:
Will the build fail if someone forgets to override this? If not, maybe it is better to use `all`